challenges/objects-arrays.js

```diff
@@ -97,29 +248,41 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
-console.log(lowerCase); 
+// const lowerCase = [];
+// console.log(lowerCase);
+
+const lowerCase = zooAnimals.map(zoo => {
+  return zoo.animal_name.toLowerCase();
+});
+console.log(lowerCase);
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
+// const lowerPopulation = [];
+
+let lowerPopulation = zooAnimals.filter(function(pop) {
+  return pop.population <= 5;
+});
+
 console.log(lowerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
-console.log(populationTotal);
 
+let populationTotal = zooAnimals.reduce(
```

Comment:
Excellent work! Your reduce function does the right thing but the readability is off. Can you spot the difference below?

```
let populationTotal = zooAnimals.reduce((total, animal) => total+ animal.population, 0);
```